src/index.js

```diff
@@ -80,6 +80,12 @@ function createPushHandler(ctx) {
 
 const wrappedFetch = async (ctx, url, options = {}) => {
   const opts = { ...DEFAULT_FETCH_OPTIONS, ...options };
+  // sanitze method name (#24)
+  if (typeof opts.method === 'string') {
+    opts.method = opts.method.toUpperCase();
+  } else {
+    delete opts.method;
```

Comment:
why delete ? wouldn't it be illegal to use a non-string method or undefined ?

Reply:
I believe lowercase method names would also be illegal